docs/manage/fleet/_index.md

```diff
@@ -56,30 +58,52 @@ Role Based Access Control (RBAC) ia a way to enforce security in the [Viam app](
+
+Robot [configuration](robots/#configuration) and robot [code](#control-with-code) is intentionally kept separate, allowing you to keep track of versioning and debug issues separately.
 
-When a robot or a {{< glossary_tooltip term_id="part" text="robot part" >}} that is managed with the Viam app first comes online, it requests its configuration from the [Viam app](https://app.viam.com).
-Once the robot has a configuration, it caches it locally and can use the configuration for up to 60 days.
-The robot checks for new configurations every 15 seconds and changes its configuration automatically when a new configuration is available.
+### Configuration
 
-{{< alert title="Tip" color="tip" >}}
-If you are managing a large fleet, you can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} when [configuring your robot](../configuration).
+Everyone who has access to the location the robot is in, can change the robot's configuration.
+When you or your collaborators change a robot's configuration, the robot will automatically reconfigure itself within 15 seconds.
+You can see configuration changes made by yourself or by your collaborators on the [History tab](robots/#history).
+You can also revert to an earlier configuration from the History tab.
+
+{{< alert title="Note" color="note" >}}
+For some configuration aspects you may require physical access to the robot so you can see how components are connected.
 {{< /alert >}}
 
-## Package Deployment
+### Package Deployment
 
 _Coming soon._
 
-Deploy control logic, [modular resources](/program/extend/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](../../services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
+You and your collaborators can deploy control logic, [modular resources](/program/extend/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](../../services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
+
+### Remote control
+
+Everyone who has access to the robot can remotely control it using the [**control** tab](robots/#control).
+This allows you to visually test and remotely operate robot components and services
+If you are remotely controlling the robot, it is recommended that you or your collaborators install a [camera](/components/camera) so you can monitor the robot as you control it.
+
+### Control with code
+
+Everyone who has access to the robot's location can obtain the robot address and secret needed to send API calls to the robot.
+You can also share the robot address and location secret without granting location access in the Viam app.
+
+With the robot address and the location secret, you can write code, collaborate on your code using tools like GitHub, and run your code on the robot from anywhere in the world.
```

Comment:
```suggestion
With the robot address and the location secret, you can write code, collaborate on your code using tools like GitHub, and run your code to control the robot from anywhere in the world.
```
As the code in some cases does not need to be running on the robot, and in order to run it on the robot they'd need access to the robot compute or code deployment... I think as written it confuses what we mean to convey here